### PR TITLE
Highlight and CH2 sectionCourier fixes

### PR DESCRIPTION
**Pull request by Shushishtok**
- Added a highlight animation for all highlights (didn't test ALL of them, but the ones I tested seem good). It will now fly from above the element with 5x the size, and will centralize itself in its actual position over 0.5 seconds.
- Added a hardcoded path and class for the Deliver Items button, which is a circle. We can do the same for Scan and Glyph, but it might not be necessary because it looked good either way.
- Fixed the Dire Secret Shop being in the incorrect location in the minimap.
- Fixed the Demon Edge being buyable twice, softlocking the player.
- Fixed Daedalus & Recipe softlocking if the courier buys the items instead of the player.
- Fixed items staying on stash/courier if skipping mid-section.
- Fixed the path arrow of the last location being weird.
- Other slight fixes to assist with the flow.

Fixes https://github.com/ModDota/dota-tutorial/issues/263

### CODE REVIEW

File: content/panorama/scripts/custom_game/hud.ts [hunk 1]
```diff
@@ -97,7 +97,12 @@ function highlightUiElement(event: NetworkedData<HighlightElementEvent>) {
 
         const highlightPanel = $.CreatePanel("Panel", $.GetContextPanel(), "UIHighlight");
         highlightPanel.hittest = false; // Dont block interactions
-        highlightPanel.AddClass("UIHighlight");
+        highlightPanel.AddClass("UIHighlightScalingAnimation")
+        $.Schedule(0.5, () => {
+            highlightPanel.RemoveClass("UIHighlightScalingAnimation")
```
Comment (RobinKa) [inline content/panorama/scripts/custom_game/hud.ts:102]: What if the panel got destroyed before this gets called?

**Review by RobinKa — COMMENTED** [review]
Looks good

File: content/panorama/scripts/custom_game/hud.ts [hunk 2]
```diff
@@ -99,8 +99,11 @@ function highlightUiElement(event: NetworkedData<HighlightElementEvent>) {
         highlightPanel.hittest = false; // Dont block interactions
         highlightPanel.AddClass("UIHighlightScalingAnimation")
         $.Schedule(0.5, () => {
-            highlightPanel.RemoveClass("UIHighlightScalingAnimation")
-            highlightPanel.AddClass("UIHighlight");
+            if (highlightPanel.IsValid())
+            {
```
Comment (RobinKa) [inline content/panorama/scripts/custom_game/hud.ts:103]: This angers me greatly

Comment (Shushishtok) [reply content/panorama/scripts/custom_game/hud.ts]: Unangered you greatly

**Pull request merged**